CHANGELOG.md

```diff
@@ -1,10 +1,40 @@
+## 0.9.0-alpha+10
+
+> NOTE: This code is considered production quality, but depends on angular:
+> 5.0.0-alpha+9. The alpha tag represents the evolving nature of the AngularDart
+> API, not code quality (5.0.0-alpha+9 is used in production Google apps).
+
+ * Add Material Slider component.
+ * Remove `visibility.all` from `@Component` annotations. This is potentially
+   BREAKING.
```

Comment:
Did you want to mention that some `preserveWhitespace: true` have been removed?

Reply:
(Agreed, but from experience, I had to adjust some of the example apps following this switch in default in Angular ... so I just thought that I'd bring it up.)